tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,204 @@
+# We can also view the epoched data for the control condition and observe
+# that it does not show the expected morphology.
+
+epochs['Control'].plot_image(combine='mean', vmin=-30, vmax=30,
+                             ts_args=dict(ylim=dict(hbo=[-15, 15],
+                                                    hbr=[-15, 15])))
+
+
+###############################################################################
+# View consistency of responses across channels
+# ---------------------------------------------
+#
+# Similarly we can view how consistent the response was across the optode
+# pairs that we selected. All the channels in this data were located over the
+# motor cortex, and all channels show a similar pattern in the data.
+
+fig, axes = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
+clims = dict(hbo=[-20, 20], hbr=[-20, 20])
+epochs['Control'].average().plot_image(axes=axes[:, 0], clim=clims)
+epochs['Tapping'].average().plot_image(axes=axes[:, 1], clim=clims)
+axes[0, 0].set_title("Control\nOxyhaemoglobin")
+axes[0, 1].set_title("Tapping\nOxyhaemoglobin")
+
+###############################################################################
+# Plot standard fNIRS response image
+# ----------------------------------
+#
+# Finally we generate the most common visualisation of fNIRS data, plotting
+# both the HbO and HbR on the same figure to illustrate the relation between
+# the two signals.
+
+evoked_dict = {'Tapping/HbO': epochs['Tapping'].average(picks='hbo'),
+               'Tapping/HbR': epochs['Tapping'].average(picks='hbr'),
+               'Control/HbO': epochs['Control'].average(picks='hbo'),
+               'Control/HbR': epochs['Control'].average(picks='hbr')
+               }
+
+# Rename channels until the encoding of frequency in channel name
+# issue is fixed
+picks = _picks_to_idx(evoked_dict['Tapping/HbO'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Tapping/HbO'].rename_channels({
+        evoked_dict['Tapping/HbO'].info['chs'][ii]['ch_name']:
+            evoked_dict['Tapping/HbO'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Tapping/HbR'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Tapping/HbR'].rename_channels({
+        evoked_dict['Tapping/HbR'].info['chs'][ii]['ch_name']:
+            evoked_dict['Tapping/HbR'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Control/HbO'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Control/HbO'].rename_channels({
+        evoked_dict['Control/HbO'].info['chs'][ii]['ch_name']:
+            evoked_dict['Control/HbO'].info['chs'][ii]['ch_name'][:-4]})
+picks = _picks_to_idx(evoked_dict['Control/HbR'].info, 'fnirs')
+for ii in picks:
+    evoked_dict['Control/HbR'].rename_channels({
+        evoked_dict['Control/HbR'].info['chs'][ii]['ch_name']:
+            evoked_dict['Control/HbR'].info['chs'][ii]['ch_name'][:-4]})
+
+color_dict = dict(HbO='r', HbR='b')
```

Comment:
Let's match `defaults.py` here:
```suggestion
color_dict = dict(HbO='#AA3377', HbR='b')
```
If these do not look good, let's update `defaults.py` and here.